### DISCUSSION TITLE
Import "nonebot.adapters.onebot.v11" could not be

### DISCUSSION THREAD

**Discussion by songdf666** [nonebot/discussions#135] [问答]
<img width="466" alt="image" src="https://user-images.githubusercontent.com/81544526/166148187-5370fc8b-9087-4d60-9d7e-066506c4a064.png">
刚刚把vscode关掉了，再重新打开就发现全部都标黄了555

**Comment by yanyongyu** [comment]
vsc右下角点击python(某某环境)按钮选择正确的环境
